lib/tld.js

```diff
@@ -141,7 +141,7 @@ tld.prototype.getRulesForTld = function getRulesForTld (tld, default_rule) {
   }
 
   // Nothing found, apply some default value
-  if (!rules) {
+  if (rules === void 0) {
```

Comment:
I am not sure to get it right, as your test relates to `.eu`, and for this same case, the `rules === void 0` syntax does not tell me what to expect in that case (going into the `if` statement or not).

To be fair, I do not see the difference between `rules === void 0` and `!rules` (which is a shortcut for `rules === null || rules === undefined`).